NIAD-1085: release 0.0.1 into main (#63)
nhsconnect/integration-adaptor-lab-results: release/DOCKERHUB.md

# Quick reference
- Maintained by: NHS Digital
- Where to get help: https://github.com/nhsconnect/integration-adaptor-lab-results
- Where to file issues: https://github.com/nhsconnect/integration-adaptor-lab-results/issues

# What is the Lab Results Adaptor?
* A pre-assured implementation of Pathology and Screening FHIR adapter

# How to use this image

To help you begin using the Lab Results Adaptor we provide shell scripts and Docker Compose configurations.

## Pre-requisites

To get running make sure you have an OpenTest environment setup. The project also includes mock dependencies for local
testing and development.

## Clone the repository

```bash
git clone https://github.com/nhsconnect/integration-adaptor-lab-results.git
```

## Pull the latest changes and checkout the release tag

> Every tagged container on Docker hub has a corresponding tag in the git repository. Checkout the tag of the release 

Reviewer: ```suggestion
Every tagged container on Docker Hub has a corresponding tag in the GitHub repository. Checkout the tag of the release 
```

you are testing to ensure compatibility with configurations and scripts.

```bash
git pull
git checkout 0.0.1
```

## Configure the application

> docker-compose.yml is preconfigured with environment values to run a standalone version of the adadptor using fake mesh.

Reviewer: ```suggestion
`docker-compose.yml` is preconfigured with environment values to run a standalone version of the adaptor using fake mesh.
```

> If you would like to run the adaptor against any other Mesh service or using any other database or message queue broker, modify required environment variables as described in to  [README](https://github.com/nhsconnect/integration-adaptor-lab-results/blob/0.0.1/README.md)

Reviewer: ```suggestion
If you would like to run the adaptor against any other Mesh service or using any other database or message queue broker, modify required environment variables as described in the [README](https://github.com/nhsconnect/integration-adaptor-lab-results/blob/main/README.md)
```

The linked README should be to the one in the main branch, not the release branch?

## Find the release directory

```bash
cd ../release
```

## Start the adaptor

The script pulls the released Lab Results adaptor container image from Docker Hub. It builds containers for its dependencies
from the Dockerfiles in the repository.

```bash
./run.sh
```

## Monitor the logs

```bash
./logs.sh
```

## Run the tests

We provide shell scripts in the release/tests directory to help you start testing.

* `healthcheck.sh` verifies that the adaptor's healthcheck endpoint is available

> * `send_message.sh` puts an example Edifact message on the fake mesh allowing it to be consumed and translated to FHIR by the adaptor

Reviewer: ```suggestion
* `send_message.sh` puts an example EDIFACT message on the fake mesh allowing it to be consumed and translated to FHIR by the adaptor
```
